data/api/client-server/pushrules.yaml

```diff
@@ -545,7 +545,7 @@ paths:
           required: true
           description: |
             The kind of rule
-          example: cake
+          example: content
```

Comment:
?

Reply:
It's an enum so it has to match one of the acceptable values.

Reply:
Oh - indeed